__tests__/app.test.js

```diff
@@ -92,6 +94,85 @@ describe("GET", () => {
         });
     });
   });
+
+  describe("/api/articles", () => {
+    test("status 200", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { message } }) => {
+          expect(message).toBe("endpoint connected successfully");
+        });
+    });
+
+    test("status 200: should return an array of article objects. Each should have property of: author, title, article_id, created_at and votes", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          articles.forEach((article) => {
+            expect(articles);
+            expect(articles).toHaveLength(12);
+            expect(article).toEqual(
+              expect.objectContaining({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+              })
+            );
+          });
+        });
+    });
+
+    test("status 404 : should return path not found when incorrect path given", () => {
```

Comment:
You only need one 404 test that should test globally instead of being tied to any one endpoint